Automattic/wp-calypso
# Happy Blocks: Split WPcom header and education header

## Proposed Changes

Currently, there's no way to load either the WpCom main nav bar or the Education header (search and title), you would always get both and do CSS magic to hide or show one of them if necessary. This PR splits them into two different blocks, allowing you to load them individually. This will make our code more reusable and decoupled. The /learn webinars and courses inner pages is an example of it, they only need the WpCom nav bar.

## Testing Instructions

- Pull and run this branch
- Navigate to `apps/happy-blocks` and run `yarn dev --sync` to sync to your sandbox
- Follow the steps written on [this PR](https://github.com/Automattic/wpsupport3/pull/486)

**On /support nothing should be changes, please check around too**
